corim/signer.go

```diff
@@ -4,35 +4,27 @@ import (
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/elliptic"
-	"errors"
 	"fmt"
 	"reflect"
 
-	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/lestrrat-go/jwx/v2/jwk"
 	cose "github.com/veraison/go-cose"
 )
 
-func getAlgAndKeyFromJWK(j string) (cose.Algorithm, crypto.Signer, error) {
+func getAlgAndKeyFromJWK(j []byte) (cose.Algorithm, crypto.Signer, error) {
```

Comment:
I did not changed the function but used the equivalent API ks.Key(0) which returns the key on Index 0 and it works, with minimal change!

Reply:
since we don't deal in keysets it looks more correct to change this to use the right api

Reply:
However the earlier implementation was using key set, isn't it?

Anyway either way it is fine, we are using the same library tools!